Makefile

```diff
@@ -182,6 +182,10 @@ vendor: $(GO)
 	$(GO) mod tidy
 	$(GO) mod vendor
 
+auto-bumper:
+	$(GO) install ${GOPATH}/src/github.com/kubevirt/cluster-network-addons-operator/tools/bumper
+	${GOBIN}/bumper ${ARGS}
```

Comment:
Since we are with go modules we can just do `go run tools/bumper ${ARGS}`

Reply:
AFAIK, since bumper is a script comprised of more than 1 file, building it is mandatory

Reply:
nope, it will compile and use them, since they are under the same project (now is when  you test it and it does not work :-).